### PR TITLE
[FOR-2024] - add Fortaleza event

### PR DESCRIPTION
**Pull request by ana-aguilar7**
## Context

[FOR-2024] - This pull request adds the DevopsDays Fortaleza event.

### CODE REVIEW

**Comment by tutagomes** [comment]
Hi @ana-aguilar7,

I wanted to highlight that the DevOpsDays BH team previously submitted a PR on February 18th #13685 for the event scheduled on September 28th, which was later updated two weeks ago to September 21st (#13943) . Can you confirm if you'll be aligning with this new date for the Fortaleza event?

 Thanks for your dedication and commitment to the community in making DevOpsDays happen.

**Comment by renat473** [comment]
Hello @tutagomes ,

Hello, in conversation with one of the organizers, our date and event will not impact the public.

Where the public is at different ends of the country, which will not cause conflict between the public or sponsors.

File: data/events/2024/fortaleza/main.yml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+location: "Local: Estácio - Universidade Estácio do Ceará - Fortaleza/CE" # Defaults to city, but you can make it the venue name.
+#
+location_address: "R. Eliseu Uchôa Beco, 600 - Guararapes" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  #- name: propose
+  - name: location
+  # - name: registration
+  #   url: https://devopsdaysce.eventize.com.br
+  - name: program
```
Comment (phrawzty) [inline data/events/2024/fortaleza/main.yml:43]: Strongly suggest disabling this link until you actually have a programme to share.
```suggestion
  #- name: program
```

File: data/events/2024/fortaleza/main.yml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+  - name: speakers
```
Comment (phrawzty) [inline data/events/2024/fortaleza/main.yml:44]: Strongly suggest disabling this link until you actually have speakers to share.
```suggestion
  - name: speakers
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2024/fortaleza/main.yml (hunk 2 above)
Comment (ana-aguilar7) [reply data/events/2024/fortaleza/main.yml]: @phrawzty done.
Can you check, please?

**Pull request merged**